README.md

```diff
@@ -239,16 +247,18 @@ Storefront.QueryRoot response = ...;
 Storefront.Collection collection = (Storefront.Collection) response.withAlias("collection").getNode();
 Storefront.Product product = (Storefront.Product) response.withAlias("product").getNode();    
 ```
+
 Learn more about [GraphQL aliases](http://graphql.org/learn/queries/#aliases).
 
 ## GraphClient [⤴](#table-of-contents)
-The `GraphClient` is a network layer built on top of `OkHttp` from Square that prepares `GraphCall` to execute `query` and `mutation` requests. It also provides conveniences for polling and retrying requests. To get started with `GraphClient ` you need the following:
 
-- Your shop's `.myshopify.com` domain.
-- Your API key, which can be obtained in your shop's admin page.
-- Optional `OkHttpClient` if you want to customize the configuration used for network requests or share your existing `OkHttpClient` with the `GraphClient`.
-- Optional settings for http cache, like path to the cache folder and maximum allowed size in bytes.
-- Optional http cache policy to be used as default for all GraphQL **query** operations (will be ignored for mutation operations as http cache is not supported for them). By default http cache policy is set to `NETWORK_ONLY`.
+The `GraphClient` is a network layer built on top of Square's [**OkHttp**](https://github.com/square/okhttp/) client that prepares `GraphCall` to execute `query` and `mutation` requests. It also simplifies polling and retrying requests. To get started with `GraphClient`, you need the following:
```

Comment:
in the iOS docs we refer to this as `Graph.Client` instead of `GraphClient` -- does that difference matter?